src/core/tests/Test_CoreMain.py

```diff
@@ -278,7 +278,7 @@ def test_assessment_operation_fail(self):
         self.assertEquals(len(substatus_file_data), 2)
         self.assertTrue(substatus_file_data[0]["name"] == Constants.PATCH_ASSESSMENT_SUMMARY)
         self.assertTrue(substatus_file_data[0]["status"].lower() == Constants.STATUS_ERROR.lower())
-        self.assertEqual(len(json.loads(substatus_file_data[0]["formattedMessage"]["message"])["errors"]["details"]), 5)
+        self.assertEqual(len(json.loads(substatus_file_data[0]["formattedMessage"]["message"])["errors"]["details"]), 3)
```

Comment:
Why did this value change?

Reply:
This value changed because of the retry logic. It was originally 2 (before any retries were added), but then set to 5 because of the 5 retries for refresh_repo and how it reported the errors. Now it is back to 2 because there are no more duplicate errors even with the new retries in the invocation.